client/config-service-rest-client/src/main/java/org/hawkular/apm/config/service/rest/client/ConfigurationServiceRESTClient.java

```diff
@@ -70,115 +62,17 @@ public ConfigurationServiceRESTClient() {
      */
     @Override
     public CollectorConfiguration getCollector(String tenantId, String type, String host, String server) {
-        if (log.isLoggable(Level.FINEST)) {
-            log.finest("Get collector configuration: tenantId=[" + tenantId + "] type=[" + type + "] host=[" + host
-                    + "] server=[" + server + "]");
-        }
-
-        // Check if configuration provided locally
-        if (PropertyUtil.getProperty(ConfigurationLoader.HAWKULAR_APM_CONFIG) != null) {
-            CollectorConfiguration ret = ConfigurationLoader.getConfiguration(type);
-            if (log.isLoggable(Level.FINEST)) {
-                try {
-                    log.finest("Collector configuration [local] = " + (ret == null ? null :
-                                    mapper.writeValueAsString(ret)));
-                } catch (Throwable t) {
-                    log.finest("Collector configuration [local]: failed to serialize as json: "+t);
-                }
-            }
-            return ret;
-        }
-
-        if (!isAvailable()) {
```

Comment:
I think this check should also be performed. This method shouldn't be called if the service impl is considered unavailable, but best to check.